### PR TITLE
Add font scaling

### PR DESCRIPTION
**Pull request by edsonayllon**
# Issue Number

#176

# Description 

Font scaling can be accessed via the footer. Increases up to 200% and decreases down to 70% size. App works as expected after scaling. Tested on both web and mobile. 

Done using existing Redux store, and CSS property `transform: scale()`.

# Screenshots

## Mobile

Screen Shot 2019-12-18 at 8 28 04 PM
Screen Shot 2019-12-18 at 8 10 44 PM

## Web

Screen Shot 2019-12-18 at 8 11 09 PM
Screen Shot 2019-12-18 at 8 10 59 PM

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -26,7 +26,15 @@ export const Footer = connect(({ authenticated, status, settings, user }) => ({
     cursorBack()
   }}>
     <li>
-
+      <span className="floatLeft">
+        <a className='increaseFont' onClick={() => {
+          if (scaleSize < 2) dispatch({ type: 'scaleSize', value: scaleSize + 0.1 })
```
Comment (raineorshine) [inline src/components/Footer.js]: Please add three constants to `constants.js`:

- `MIN_FONT_SIZE`
- `MAX_FONT_SIZE`
- `FONT_SCALE_INCREMENT`

Comment (edsonayllon) [reply src/components/Footer.js]: Added in 60ff4d7

**Comment by edsonayllon** [comment]
> @raineorshine The GestureDiagram is an svg component that must be explicitly scaled, as it does not scale with the container.

Hmm. It seems to be scaling for me. On iPhone 8:

Screen Shot 2019-12-20 at 4 38 57 PM
Screen Shot 2019-12-20 at 4 39 09 PM

I've tested using `scale()` elsewhere, and it also scales SVGs inside of the container being scaled.  `Create-React-App`, where spinning logo is an SVG:

Screen Shot 2019-12-20 at 4 51 16 PM
Screen Shot 2019-12-20 at 4 51 28 PM

It doesn't scale the diagrams within `/help`, but scaling doesn't pass to there, so that entire section is untouched. 

I tested manually passing calculated sizes anyways, and it resulted in these triangles (I changed the color to red to find the diagram in the app).

Screen Shot 2019-12-20 at 4 11 34 PM

So, passing through new sizes may not be a good idea, and isn't needed.

**Comment by raineorshine** [comment]
You are right. My apology! I think the default sizing is off, which is what made me think it wasn't scaling. Looks great then. Thank you!

**Pull request merged**